source/includes/inkling-reference/_curriculum.html.md

```diff
@@ -152,31 +170,49 @@ with data
 
 Any simulator or generator referenced in a curriculum must have an associated simulator or generator clause.
 
-## Simulator / Generator Clause Syntax
+## Simulator Clause Syntax
+
+When a simulator is used for training, use the simulator clause.
 
-```c
-simulator <simulatorName>'('<configurationSchema>')' // simulator clause
-  state '('<stateSchema>')' // simulator state
-  control '('<controlSchema>')' // training concept predicts schema
+The keyword **simulator** indicates that the responses of the system to the training
+data will feed back into the training. Simulators are coded implementations of
+the lessons. They are time variant. A simulator coded for example in python
+keeps state and thus it will use deep-q learning. The simulator clause has a
+state schema that describes simulator state.  
+
+```plaintext
+simulator <simulatorName>'('<configurationSchema>')' 
+  state '('<stateSchema>')'     # simulator state
+  control '('<controlSchema>')' # training concept predicts schema
 end
 ```
+## Generator Clause Syntax
+
+When a generator is used for training, use the generator clause.
 
-```c
-generator <generatorName>'('<configurationSchema>')'  // generator clause
-  yield '('<outputSchema>')'    // generator output (yield)
+The keyword **generator** indicates that the simulator generates the training data.
+Generators can be thought of as a stateless simulators. They do have coded
+simulators but the training output does not get fed back into simulator. The
+generator clause has a yield schema that defines the training output.
+
+```plaintext
+generator <generatorName>'('<configurationSchema>')'  
+  yield '('<outputSchema>')'    # generator output (yield)
 end
 ```
 
-###### Curriculum Examples
+## Curriculum Example
```

Comment:
The intent is to use 'Example' for a single example and Examples where there is more than one. I fixed one reference to that